bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
         </dd>
       </dl>
 
-      <h2>Bug reporting</h2>
+      <h2 id="bug-reporting">Bug reporting</h2>
 
       <dl class="faq">
+        <dt id="what-next">Once I have found a vulnerability, what next?</dt>
+        <dd>
+          <p>The sooner we can reproduce and fix the bug, the sooner we can
+            fix it and send you your bounty. Please submit all bugs through
+            the <a href="https://bugzilla.mozilla.org/form.web.bounty">Bugzilla web bounty form</a>.
+            <strong>Do not send vulnerabilities via email and please avoid using video.</strong></p>
+
+          <p>There are three main things you can provide which will help us
+            to evaluate your submission quickly and pay a bounty sooner:</p>
+
+          <ol>
+            <li><a href="{{ url('security.bug-bounty.faq-webapp') }}#attack-scenario">What is the attack scenario?</a></li>
+            <li><a href="{{ url('security.bug-bounty.faq-webapp') }}#step-by-step-exploit">What is the step-by-step exploit process?</a></li>
```

Comment:
Nit-pick here, but no need for the full URL if we're just linking to an in-page anchor.

Reply:
Good call, I forgot to clean that up when I moved this text from a different page.